platform-core-module/src/main/java/com/softicar/platform/core/module/file/stored/server/AGStoredFileServer.java

```diff
@@ -13,6 +15,20 @@ public IDisplayString toDisplayWithoutId() {
 		return IDisplayString.create(getUrl());
 	}
 
+	public static List<AGStoredFileServer> getAllActiveWithPrimaryFirst() {
+
+		List<AGStoredFileServer> servers = new ArrayList<>();
+		Optional<AGStoredFileServer> primaryServer = getInstance();
+		getInstance().ifPresent(servers::add);
+		AGStoredFileServer.TABLE//
```

Comment:
Could be expressed as a single statement, via SQML. Probably not worth the hassle though. Just sayin.